core/views.py

```diff
@@ -143,3 +145,20 @@ def coc(request, lang=None):
         return render(request, template_name)
     except TemplateDoesNotExist:
         raise Http404("No translation for language {}".format(lang))
+
+
+def crowdfunding_donors(request):
+    donor_list = Donor.objects.filter(visible=True).order_by('amount')
```

Comment:
Should we add secondary order by? I.e. name? Django by default orders ascending, so order `amount` will return smaller values first. I think it makes sense to display bigger donors first, right?

Like: `order_by('-amount', 'name')`